### PR TITLE
codespell: workflow, config + typos fixed

### PR DESCRIPTION
**Pull request by yarikoptic**

### CODE REVIEW

File: tests_old/test_university.py [hunk 1]
```diff
@@ -77,9 +77,9 @@ def test_restrict():
     assert_true(len(s1) == 11)
     assert_list_equal(s1, s2)
 
-    millenials = Student & 'date_of_birth between "1981-01-01" and "1996-12-31"'
-    assert_true(len(millenials) == 170)
-    millenials_no_math = millenials - (Enroll & 'dept="MATH"')
+    millennials = Student & 'date_of_birth between "1981-01-01" and "1996-12-31"'
+    assert_true(len(millennials) == 170)
+    millenials_no_math = millennials - (Enroll & 'dept="MATH"')
     assert_true(len(millenials_no_math) == 53)
```
Comment (dimitri-yatsenko) [inline tests_old/test_university.py:83]: also in this variable then.

Comment (yarikoptic) [reply tests_old/test_university.py]: thanks for spotting!  fixed that one as well in the last commit (ec46b787ce121385ae533568eb49eac1df47ae4d)

**Comment by yarikoptic** [comment]
done

**Review by dimitri-yatsenko — APPROVED** [review]

**Comment by dimitri-yatsenko** [comment]
I don't believe this requires an entry in the CHANGELOG. Ready to merge.

**Review by kabilar — APPROVED** [review]
Thanks @yarikoptic!

**Comment by kabilar** [comment]
> I don't believe this requires an entry in the CHANGELOG. Ready to merge.

Thanks @dimitri-yatsenko.  I agree.

**Pull request merged**